cpp/test/prims/batched/matrix.cu

```diff
@@ -208,7 +208,7 @@ class MatrixTest : public ::testing::TestWithParam<MatrixInputs<T>> {
         break;
       case Hessenberg_op: {
         constexpr T zero_tolerance =
-          std::is_same<T, double>::value ? 1e-8 : 1e-4f;
+          std::is_same<T, double>::value ? 1e-7 : 1e-3f;
```

Comment:
Is there any benefit to keeping the original margins commented out for reference?

Reply:
I think it might be a better idea to keep a list of the precise changes in the github issue #1949, will add them in a bit